### PR TITLE
feat: card series filter, fix: newegg `oosLabels`

### PR DESCRIPTION
**Pull request by admon84**
### Description

* Adding filterable card series (debug, 3080, and eventually 3090)
* Added 'debug' cards for validating configuration works for "in stock" cards (don't want to miss a chance because the config was wrong - looking at you newegg config)
* Moved oosLabels and captchaLabels to be store props instead of in each link (since they are always the same)

### Testing

* Tested 'debug' and '3080' series on all stores
* Ran lint script

### New dependencies

N/A

### CODE REVIEW

**Comment by admon84** [comment]
Newegg oosLabel for `'out of stock'` was throwing "out of stock" message even for in-stock cards.  When I noticed that I figured the "debug" series could be used for testing in-stock cards for each store and make sure it says "IN STOCK" when it should.

File: src/config.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ const page = {
 };
 
 const store = {
+	showOnlySeries: process.env.SHOW_ONLY_SERIES ? process.env.SHOW_ONLY_SERIES.split(',') : ['3080'],
```
Comment (Doridian) [inline src/config.ts:62]: I'd default to `[]` here, same as for brands to defaul to "show all" (or maybe `['3070','3080','3090']` to hide debug cards)?

**Review by Doridian — DISMISSED** [review]
Approved with comment.

File: src/config.ts (hunk 1 above)
Comment (admon84) [reply src/config.ts:62]: Yeah, let me change to this `['3070','3080','3090']` as the default.  Using `[]` will include the debug cards and probably not a great idea if someone messes up their environment config

**Comment by Doridian** [comment]
Also please fix lint errors. Looks good otherwise.

**Review by Doridian — DISMISSED** [review]

File: src/store/includes-labels.ts [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+ * Checks if DOM has any related text.
  *
  * @param domText Complete DOM of website.
- * @param oosLabels Out-of-stock labels.
+ * @param searchLabels Search labels for a match.
  */
-export function includesLabels(domText: string, oosLabels: string[]): boolean {
+export function includesLabels(domText: string, searchLabels: string[]): boolean {
 	const domTextLowerCase = domText.toLowerCase();
-	return oosLabels.some(label => domTextLowerCase.includes(label));
```
Comment (jef) [inline src/store/includes-labels.ts:9]: I think this was fine the way it was written. Any reason for the change?

**Review by jef — CHANGES_REQUESTED** [review]
Other than the one comment, I really like what you did here!

Thank you!

File: src/store/includes-labels.ts (hunk 2 above)
Comment (admon84) [reply src/store/includes-labels.ts:9]: Agreed, it was fine the way it was.  It was a remainder of another update that got away from me.

Comment (admon84) [reply src/store/includes-labels.ts:9]: I reverted it back to using `array.some()`. Thanks for the feedback

Comment (jef) [reply src/store/includes-labels.ts:9]: Ahh, no problem. Just curious if we goofed something up. No biggie. Thanks!

**Review by jef — APPROVED** [review]

**Comment by Ellementz** [comment]
By model would be super nice FTW3 anyone?

**Pull request merged**